### DISCUSSION TITLE
How to get coverage for .vue files?

### DISCUSSION THREAD

**Discussion by Hyphon** [vitest-dev/vitest#328] [Q&A] [answered]
when i run `vitest --coverage` in vitest/test/vue，
i got result like:
![image](https://user-images.githubusercontent.com/11419600/147442574-97c4ac4e-4c12-47aa-9a2a-40cf97103191.png)
i tried to create a c8 configuration file, and set "include" property, but still cant work

**Comment by farazshuja** [comment, marked as answer]
I was facing the same but its now showing coverage after I upgrade to node v.14.18.3 (earlier it was v.14.17.x)

**Reply by Hyphon** [reply]
bump vitest to 0.2.8, and everything is Ok now

**Comment by wesley-spencer** [comment]
I am running Node v16.14.0 and this still is not working, Any chance @farazshuja @Hyphon you could share your vie.config file on the thread for people who are still experiencing this issue?

**Reply by Hyphon** [reply]
```
/// <reference types="vitest" />
import { defineConfig } from 'vite'

export default defineConfig({
    ...otherConfig,
    test: {
        global: true,
        environment: 'happy-dom'
    }
})
```

node: v16.13.1
vite: v2.7.13
vitest: v0.2.8

**Reply by wesley3295** [reply]
dang I guess my problem is a Nuxt issue
